source/standards/configuration-management.html.md.erb

```diff
@@ -0,0 +1,24 @@
+---
+title: Configuration Management
+expires: 2018-05-16
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+When running VMs, such as in Amazon EC2, where the complexity of the configuration
+in the instance becomes too great for managing during deployment or within AMIs,
+use configuration management.
+
+Using configuration management ensures that the configuration of an environment is
+documented as code.
```

Comment:
We should explicitly state that this code should be treated as per other code and kept in source control, with some statement about paying special attention to any sensitive/secret data (either managing secrets entirely separately, or if no suitable mechanism exists perhaps storing them alongside the code, ideally encrypted)

Reply:
Missed this comment before :+1:'ing this PR. We shouldn't duplicate the content from the `how to store source code` section though. Maybe we could add a link to that?